en_US/voicemail_timezones.txt

```diff
@@ -6,9 +6,11 @@ voicemail_timezones.delete = Delete
 voicemail_timezones.general = General
 voicemail_timezones.none = None
 voicemail_timezones.timezone = Time Zone
-voicemail_timezones.timezone.tooltip = Timezone to be selected.
+voicemail_timezones.timezone.tooltip = Timezone to be selected
 voicemail_timezones.tzdefinition = Time Definition
+voicemail_timezones.tzdefinition.tooltip = The style of the announcement that will used when announcing voicemails. The time definition can include any sound prompt, by enclosing the name of the prompt in single quotes, as well as any date and time formatting symbols:<br>
```

Comment:
This symbol is not present in the system therefore it won't be displayed. System changes are required to add it.